### PR TITLE
replace gif status indicators with unicode symbols

### PR DESCRIPTION
**Pull request by DonKult**
Hi,

First of: Happy new year & thanks for this interesting project!

This PR drops the 3 status-indicating gifs in favor of the three unicode symbols ⭮, ✔ and ✘ which seem to be in the later two cases exactly what the gifs showed with the bonus of scaling nicely with the size and always having the right background color (which is the actual reason for this otherwise relatively pointless change). A bit of CSS is applied to make the arrow spin and apply some color.

Best regards

David Kalnischkies

### CODE REVIEW

**Comment by ohwgiles** [comment]
Hi, thanks for contributing!

I'd like to use unicode symbols for the status icons if possible. In fact, I used to use ✔ and ✘, but they behaved inconsistently with the spinner, so I replaced them with gifs back in ea133382b0.

The issue is consistency. Actually I first read your PR on my phone, and iOS displayed the open circle arrow as a bunch of horizontal lines, like the hamburger icon but more. I didn't understand how it was supposed to represent progress until I opened the PR on my desktop.

Furthermore, colours don't work on iOS unless you set the font (as I did before) to 'Zapf Dingbats' - see [this issue](https://stackoverflow.com/questions/20804637/html-unicode-arrow-works-on-safari-desktop-but-not-safari-for-ios)

Basically I am in favour of this but it needs some tweaks to work well on the most common platforms before I can merge it

**Comment by DonKult** [comment]
Thank god unicode is a standard so "U+2B6E CLOCKWISE TRIANGLE-HEADED OPEN CIRCLE ARROW" will never look like a hamburger… of course, it would have been to easy. 🗘 ↻ ⟳ ⭮

Amended with the font, color, text-forcing (and changed spinner) just because but I understand that this isn't going to work for now – and sorry I missed the commit changing to the current gifs. Thanks also for quick reviewing and the suggestions!

**Comment by ohwgiles** [comment]
OK I took a closer look at these symbols on a few different environments and while ↻ can work, I found it too thin compared to ✔ and ✘, and setting it to `bold` looks strange. I spent a while on [this page](http://www.fileformat.info/info/unicode/category/So/list.htm) and I think my favourite one that works and doesn't cause a change in height is [GEAR](http://www.fileformat.info/info/unicode/char/2699/index.htm). What do you think?

The only issue is that it appears ever so slightly wider than ✔/✘. This can be fixed by putting all the status symbols (no need for the "connecting" one) in a `<span>` with `display:inline-block` and `width:1em`, which would be a good thing to do anyway in case of any further unexpected rendering differences between systems.

Some small extra suggestions:
- It turns out that `Zapf Dingbats` is not needed if you add [U+FE0F](http://www.fileformat.info/info/unicode/char/fe0e/index.htm) as you did, which is a better solution.
- `blue` seems too stark for the spinner. Maybe `darkcyan`, `indigo`, `darkslategray` or `steelblue`?
- I know I had it before, but now `crimson` seems too pink for the failed status. I think I prefer one of `firebrick` or `orangered`
- `dimgray` for "aborted"?

**Comment by DonKult** [comment]
Thanks for looking into this & here you go with another rebase.

I am not too much into picking colors, so `blue` originated more from the spinner-likes seen in e.g. Firefox and could easily be any other color including `steelblue` (in fact, any color with some blue-percentage qualifies as "blue" in by book, same for the other colors).

Not sure if/then aborted can actually happen as I can't find related code, but from the word alone it sounds for me like it should be a more active color than grey through, so I picked indigo now (also, because travis-ci uses it for bad things if I remember right)

On the other hand, the webui uses bootstrap, so perhaps the [alert colors](https://getbootstrap.com/docs/4.0/components/alerts/) should be used instead to fit in.

btw: In case the spinning is slightly off-center: [Seems like browsers tend to have some problems](https://github.com/FortAwesome/Font-Awesome/issues/671). This started as a simple "how hard can it be…", I slowly get an idea of how hard…  :wink:

**Comment by ohwgiles** [comment]
Thanks, this looks great.

Regarding `aborted` - you are right, it actually cannot happen just now, but it is a feature I intend to implement in the not-too-distant future. Indigo is fine.

You have a point with the bootstrap alert colors, but the status icons are not the only place where it deviates from that, so if this were to be changed it makes sense to do it separately. I'm also not that committed to bootstrap and at some point I imagine laminar should get a UI makeover, so I don't see that as a big issue for now.

I didn't see any off-center spinning yet. I'll check a few more browsers but basically I'm ready to merge this

**Comment by DonKult** [comment]
Firefox 57.0.3 and 58.0~b14 (Debian unstable amd64) has a minimal wobble while Chromium 63.0.3239.84 (same system) has it slightly larger – at least with the fonts & sizes I end up with here. I am fine with it (spinning is optional eyecandy anyhow), but wanted to note it in case it comes up later.

I can't test with other systems as I don't have any, so I am very interested in your results!

**Comment by ohwgiles** [comment]
I can reproduce the problem. It's very noticeably off-center in all of
- Chromium 64.0.3239, Arch Linux
- Firefox 57.0.4, Arch Linux
- Mobile Safari 11, iOS 11.2.1
- Google Chrome 63.0.3239, Windows 10
- Edge 38.14393, Windows 10

At least for me, the following change makes it either perfect or barely noticeable on all of those browsers:
From `span.status` remove
```
height: 1em;
vertical-align: middle;
line-height: 0.75em;
```
and add
```
font-family: sans-serif;
```

**Comment by DonKult** [comment]
I still have a bit of a wobble in both browsers, but minimal (and in case of Chromium a lot reduced to before), so good find!

**Comment by ohwgiles** [comment]
Merged. If you discover any further improvements please do submit another PR. The behaviour seems to be connected to the font metrics, maybe it can be worked around by explicitly specifying a "working" font rather than relying on the user's default `sans-serif`. But that's another rabbit-hole... for now it's Good Enough.
